### ISSUE TITLE
Support of semantic scholar?

### ISSUE THREAD

**Issue by scfrank-pri** [joostkremers/ebib#308]
Ebib provides excellent support for crossref. Could this be extended to semantic scholar? Thanks for consideration.

**Comment by scfrank-pri** [comment]
I was hoping for a command similar to `biblio-crossref-lookup`. This is **not** part ob ebib, but from the output generated by biblio I can import citations into ebib with `e`  - very handy.  Theres seems to be emacs support for importing bibliographic information from semantic scholar in [this](https://github.com/akshaybadola/ref-man) project, but I cannot judge it since I use ebib and not ref-man.

**Comment by joostkremers** [comment]
I agree it would be great to have support for Semantic Scholar, but it seems it would require some work. The biblio support was fairly easy, because it only had to connect `biblio.el` to Ebib, and `biblio.el` already comes with access points for external applications.

Something similar to `biblio.el` would have to be developed for Semantic Scholar. The ref-man project you point to would be a good place to start, but I suspect it's unlikely code from `ref-man-ss.el` can be used as-is.

I'd be happy to accept a PR, but I currently don't have the time to work on this myself, unfortunately.

**Comment by scfrank-pri** [comment]
Thanks for your explanation. I can live without semantic scholar and I do not really miss anything in ebib. But today my emacs 31.0.50 tells me that I should delete the obsolete packages "biblio-core, biblio". I us a lot the command "ebib-biblio-import-doi" - so, could you provide some guidance whether this is obsolete?

**Comment by joostkremers** [comment]
If you use `ebib-biblio-import-doi`, you shouldn't delete those packages. Emacs tries to keep track of which packages the user actively installed and which were installed as dependencies of some other package.

So it seems that `biblio-core` and `biblio` were installed as dependencies of some other package, which either was deleted recently or a recent update of that package no longer declares `biblio` as a dependency.

I guess a third option is a bug in `package.el`, causing the packages to be considered obsolete even though they are not.

The easiest way to get rid of the message, I suspect, is to remove both packages and then reinstall them. (You'll only need to install `biblio`, since it'll install `biblio-core` as a dependency).

Ebib does not declare `biblio` as a dependency, because its use is optional (if you don't use biblio, you can still use Ebib), and `package.el` doesn't have a concept of optional dependencies.

Hope that clears things up a bit.

**Comment by scfrank-pri** [comment]
Thanks for the explanation, I will proceed accordingly.
